docs/contributing.rst

```diff
@@ -82,6 +82,12 @@ To run a basic set of tests that neither require ``aleth`` nor ``libz3``, run
 ``./scripts/soltest.sh --no-ipc --no-smt``. This script will run ``build/test/soltest``
 internally.
 
+.. note ::
+
+    Those working in a Windows environment wanting to run the above basic sets without aleth or libz3 in Git Bash, you would have to do
```

Comment:
Thanks for the addition! Could you compress the paragraph a little? Also I think that people should know themselves that you have to build the tests first and can translate the `/` and `\` path notations - shouldn't `/` work on windows in general nowadays?

Reply:
@chriseth to answer your windows-path-seperator question, the Windows Command Prompt only knows back-slash (`\`) but the PowerShell accepts both (`/` and `\`).

Reply:
@chriseth Quick clarification - by compressing, do you want me to make this less wordy or remove spacing? I can also eliminate the command prompt line and just state to be mindful of slashes, because as @christianparpart said, depending on where you run them, certain slash directions don't work.